### DISCUSSION TITLE
The Vue Ecosystem

### DISCUSSION THREAD

**Discussion by neneodonkor** [vue-vine/vue-vine#94] [General]
I stumbled on this a couple of days ago, and it's a very interesting solution. My question is considering Vue users are very dogmatic with SFC, how do hope to get them to consider this as a viable option? Or are you targeting React Devs who would like to use Vue?

**Comment by ShenQingchuan** [comment]
Hi, @neneodonkor 

Thanks for your interest and enthusiasm, Vue Vine is still in the early stages of development, and we need to gradually fill in the missing parts of the development experience. In fact, Vue Vine is a completely independent solution from SFC, and can even coexist with SFC within the same project. We are not committed to pushing Vue Vine into Vue's standards, but rather to making it available to the community as an interesting third-party solution.

As for the larger vision, I wonder if it would be possible to provide a file form named `.tsv` (align with `.tsx`) specifically to support this kind of syntax we've defined, but I'm skeptical and on the wait and see myself at this point.

**Reply by neneodonkor** [reply]
Oh okay. I understand your motivation. I think it is nice to see it as a solution. It enriches the ecosystem. I hope your project succeeds, and that more people will find it very useful.
